### ISSUE TITLE
Copying Vocabulary from other database schema

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/JCdmBuilder#8]
In the Vocabulary tab there is the option to copy the vocab from another schema. It is not clear to me how this works. I guess this schema should then be in the same Database since there are no separate credentials, server location asked? Can you explain in what situation this is useful?

In the first Location tab the database name is for Postgres the schema (as shown by the hint). Is this copying of vocab only possible in SQL server maybe? If so it may be a good idea to grey out the option if PostgreSQL is selected on the Location tab.

**Comment by schuemie** [comment]
The option works on Postgres but yes, the vocab schema needs to be in the same database as the CDM schema. 

I have only one database on my server with many schemas. I guess you have many databases with one schema per database?

**Comment by PRijnbeek** [comment]
Correct. They are on the same Postgres server but in different databases.

SYNPUF
     CDM
     Results

IPCI[version]
    CDM
    Results

This keeps the results better organized I think and you can run a script by changing the database name (like changing the schema in JnJ case).

I could also add a vocab schema per database in this way instead of putting it into CDM or giving them all different names

Not sure what is the general preferred approach is though, there are advantages to both options.
